packages/layout/src/components/cards/insurer/insurer.tsx

```diff
@@ -73,12 +73,14 @@ export const InsurerCard: React.FC<InsurerProviderProps> = React.memo(
 						>
 							<Toolbar
 								buttonLeft={() => (
-									<CardMainHeadingTitle title={i18n.preview.buttons.one} />
+									<CardMainHeadingTitle
+										title={current.context.insurer.organisationName}
+									/>
 								)}
 								buttonRight={RemoveButton}
 								complete={isComplete(current.context)}
 								subtitle={() => (
-									<Subtitle main={current.context.insurer.organisationName} />
+									<Subtitle secondary={i18n.preview.buttons.one} />
```

Comment:
I will try to change that in a different PR because it involves almost all cards where the main heading now is the organisation name and the previous value for "buttons.one" now is part of the 'subtitle'.